modules/ROOT/pages/install-self-managed-network-configuration.adoc

```diff
@@ -42,12 +42,25 @@ To function correctly, Runtime Fabric requires the following hostname configurat
 * US control plane: `transport-layer.prod.cloudhub.io`
 * EU control plane: `transport-layer.prod-eu.msap.io` | Runtime Fabric message broker for interaction with the control plane.
 | 443 (v1.8.50 and later)| TCP (Lumberjack) a| 
-* US control plane: `dias-ingestor-router.us-east-1.prod.cloudhub.io`
-* EU control plane: `dias-ingestor-nginx.prod-eu.msap.io`
-* EU control plane: `dias-ingestor-router.eu-central-1.prod-eu.msap.io` | Anypoint Monitoring agent for Runtime Fabric.
+* US control plane: +
+`dias-ingestor-router.us-east-1.prod.cloudhub.io` +
+`us1.ingest.mulesoft.com/api/v2/lmetrics` +
+`us1.ingest.mulesoft.com/api/v2/metrics` +
+`us1.ingest.mulesoft.com/api/v2/traces` +
+`us1.ingest.mulesoft.com/api/v2/logs` +
```

Comment:
The hostname should be sufficient here so we can keep one hostname us1.ingest.mulesoft.com and remove path name `/api/v2/...`. CC @hegderohit89 @michael-lee-mulesoft